### ISSUE TITLE
Completions doesn't work when querying a stream

### ISSUE THREAD

**Issue by rdhananjaya** [ballerina-platform/ballerina-lang#29488]
**Description:**
```ballerina
    stream<string[], io:Error> csvStream = check openReadableCsvFile.csvStream();

    stream<R, error> s = from var row in csvStream
                         where row.length() > 3
                         select arrayToR(row);
```

![Screenshot 2021-03-23 at 17 21 45](https://user-images.githubusercontent.com/1496990/112145774-ccdf3180-8c00-11eb-9dc5-07b3cba1577c.png)

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by IMS94** [comment]
Relates to #26861

**Comment by IMS94** [comment]
Checked on latest master. This has been fixed with https://github.com/ballerina-platform/ballerina-lang/pull/29386/.

**Comment by nadeeshaan** [comment]
> Checked on latest master. This has been fixed with #29386.

Verified in the master. Hence closing the issue
